wakaama-temp/Makefile

```diff
@@ -0,0 +1,63 @@
+# name of your application
```

Comment:
Why is this application not in `examples/`?

Reply:
It's not in main repo examples because there already is a wakaama example. That example had a light sensor resource initially, but we removed it to trim down the scope of the PR. I expect @leandrolanzieri to create a follow on PR for the light sensor at some point.

So, this PR seemed like a variation on that theme. In a general sense it _is_ a useful application to measure and report on temperature. Also, with the combination of SAUL and LwM2M, it seemed like a larger chunk of code that belonged with applications rather than examples.

Reply:
Arghs sorry. I got here from my MUA, not via GitHub, so I thought it was in the main repo. Sorry for the noise ^^"

Reply:
No worries, @miri64. Please also dismiss the block if it's no longer valid.